pytest_itde/__init__.py

```diff
@@ -165,6 +165,7 @@ def start_db(name, itde, exasol, bucketfs):
             environment_name=name,
             database_port_forward=exasol.port,
             bucketfs_port_forward=bucketfs_url.port,
+            ssh_port_forward=22, # TBC: which port should be used here?
```

Comment:
not 22 (reserved port, only root should use it), probably one of 2002, 2222, 20002, 22222, ..., 20002 is most likely is preferred

Reply:
I wonder, how the value for the other ports is retrieved, e.g. `database_port_forward=exasol.port`.

Reply:
1. Pytest Hook for adding command Line Options https://github.com/exasol/integration-test-docker-environment/blob/a2eb031254f6dd10f59c28d610b03486737d4b46/pytest_itde/__init__.py#L228
2. Adding command Options groups https://github.com/exasol/integration-test-docker-environment/blob/a2eb031254f6dd10f59c28d610b03486737d4b46/pytest_itde/__init__.py#L218
3. Command Option Groups Definition https://github.com/exasol/integration-test-docker-environment/blob/a2eb031254f6dd10f59c28d610b03486737d4b46/pytest_itde/__init__.py#L9
4. Fixtures that create config classes from command line options https://github.com/exasol/integration-test-docker-environment/blob/a2eb031254f6dd10f59c28d610b03486737d4b46/pytest_itde/__init__.py#L103
5. Implementation of config classes and options and option groups
6.

Reply:
Following changes are necessary. Currently, we have a option group exasol for the DB, I fear the name is not well chosen. Maybe rename to exasoldb or db.
We need also a new option group, config class, fixture for ssh. We need to store Port and later user and public key. We need to add user everywhere else to, but probably better in a extra ticket.
The ssh config needs to be also stored in the test config.

Reply:
> but probably better in a extra ticket.

Yes I agree.

Reply:
As aligned: Please see tickets 
* #343
* #345 
* #346 
* #347 

In the current PR and the current line of code I will used a constant value of 2002.
Before next release we will implement #343 to Add port, keys, user to DatabaseInfo with parent field ssh_info.
Ticket #345 will add dedicated configuration options for SSH user, port and key.